docs/azure/design-build-deploy/sample-code.md

```diff
@@ -0,0 +1,16 @@
+# Azure Landing Zone Sample Code
+
+Last updated: **{{ git_revision_date_localized }}**
+
+We have a centralized repository ([Azure Landing Zone Samples](https://github.com/bcgov/azure-lz-samples)) of sample code that you can use to get started. This repository contains sample code for application patterns and tools for use in the OCIO Azure Landing Zone.
+
+## Sample Applications
+<!-- TODO: Update these links once the sample app code is migrated to the new repo -->
+* Container-based application: [Azure Startup Sample Application (Containers)](https://github.com/bcgov/azure-startup-sample-app-containers)
+* Serverless-based application: [Azure Startup Sample Application (Serverless)](https://github.com/bcgov-c/ecf-azure-startup-sample-app-serverless)
```

Comment:
Do you think we can move these to the new azure samples repo?

Teams don't have access to bcgov-c so we shouldn't link that

Reply:
Do we want to move them as-is, even though the sample apps probably won't actually work/deploy in the environment (now that we have PaaS policies in place)?

Reply:
Removed reference to Sample Apps until the code can be updated to work in the Landing Zones environment (with the latest security policies in place)